cmd/tsdb/main.go

```diff
@@ -31,19 +31,35 @@ import (
 	"time"
 
 	"github.com/go-kit/kit/log"
+	"github.com/m3db/m3/src/cluster/shard"
+	"github.com/m3db/m3/src/dbnode/persist/fs"
+	"github.com/m3db/m3/src/dbnode/retention"
+	"github.com/m3db/m3/src/dbnode/sharding"
+	"github.com/m3db/m3/src/dbnode/storage"
+	"github.com/m3db/m3/src/dbnode/storage/namespace"
+	"github.com/m3db/m3/src/dbnode/storage/series"
+	"github.com/m3db/m3x/ident"
+	xtime "github.com/m3db/m3x/time"
 	"github.com/pkg/errors"
 	"github.com/prometheus/tsdb"
 	"github.com/prometheus/tsdb/labels"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+var (
+	defaultM3DBNamespace = ident.StringID("defaultM3DBNamespace")
+	defaultSeriesID      = ident.StringID("default_m3db_id")
+	defaultNumShardsM3DB = 1
```

Comment:
I would set this to `runtime.NumCPU()` to reduce contention as much as possible.

Reply:
Oh I already updated that, let me push the latest branch (turns out it makes very little diff)

Reply:
I see, yeah looks like most of the IDs are pretty stable? i.e. not a lot of insertions after startup.